app/controllers/welcome_controller.rb

```diff
@@ -0,0 +1,5 @@
+class WelcomeController < ApplicationController
+  def index
+    
```

Comment:
No need to include a welcome on this IC unless you want to! Feel free to skip it if you'd like to save time

Reply:
I see, so my first route can have the "/" in it? I felt as though that was the issue when I first started.

Reply:
All paths should start with a "/"
There's no requirement to have a root route ("/" alone) set up, especially in an IC. You can just start with whatever routes are in the user stories!

Reply:
thank you!